{
	"name": "CVE-2024-39329",
	"threat_severity": "Low",
	"public_date": "2024-07-09T14:00:00Z",
	"bugzilla": {
		"description": "python-django: Username enumeration through timing difference for users with unusable passwords",
		"id": "2295936",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2295936"
	},
	"cvss3": {
		"cvss3_base_score": "3.7",
		"cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:L/I:N/A:N",
		"status": "draft"
	},
	"cwe": "CWE-208",
	"details": [
		"An issue was discovered in Django 5.0 before 5.0.7 and 4.2 before 4.2.14. The django.contrib.auth.backends.ModelBackend.authenticate() method allows remote attackers to enumerate users via a timing attack involving login requests for users with an unusable password.",
		"A vulnerability was found in Python-Django in the django.contrib.auth.backends.ModelBackend.authenticate() method. This flaw allows remote attackers to enumerate users via a timing attack involving login requests for users with unusable passwords."
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2024-39329\nhttps://nvd.nist.gov/vuln/detail/CVE-2024-39329"
	],
	"mitigation": {
		"lang": "en:us",
		"value": "Mitigation for this issue is either not available or the currently available options don't meet the Red Hat Product Security criteria comprising ease of use and deployment, applicability to widespread installation base or stability."
	},
	"package_state": [
		{
			"cpe": "cpe:/a:redhat:ansible_automation_platform",
			"fix_state": "Affected",
			"package_name": "ansible-tower",
			"product_name": "Red Hat Ansible Automation Platform 1.2"
		},
		{
			"cpe": "cpe:/a:redhat:ansible_automation_platform:2",
			"fix_state": "Affected",
			"package_name": "ansible-lightspeed-container",
			"product_name": "Red Hat Ansible Automation Platform 2"
		},
		{
			"cpe": "cpe:/a:redhat:ansible_automation_platform:2",
			"fix_state": "Affected",
			"package_name": "automation-controller",
			"product_name": "Red Hat Ansible Automation Platform 2"
		},
		{
			"cpe": "cpe:/a:redhat:ansible_automation_platform:2",
			"fix_state": "Affected",
			"package_name": "python-django",
			"product_name": "Red Hat Ansible Automation Platform 2"
		},
		{
			"cpe": "cpe:/a:redhat:certifications:1::el7",
			"fix_state": "Affected",
			"package_name": "python-django",
			"product_name": "Red Hat Certification for Red Hat Enterprise Linux 7"
		},
		{
			"cpe": "cpe:/a:redhat:certifications:1::el8",
			"fix_state": "Affected",
			"package_name": "redhat-certification",
			"product_name": "Red Hat Certification for Red Hat Enterprise Linux 8"
		},
		{
			"cpe": "cpe:/a:redhat:certifications:1::el9",
			"fix_state": "Affected",
			"package_name": "redhat-certification",
			"product_name": "Red Hat Certification for Red Hat Enterprise Linux 9"
		},
		{
			"cpe": "cpe:/a:redhat:discovery:1.0::el8",
			"fix_state": "Affected",
			"package_name": "discovery/discovery-server-rhel8",
			"product_name": "Red Hat Discovery"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:16.1",
			"fix_state": "Affected",
			"package_name": "python-django20",
			"product_name": "Red Hat OpenStack Platform 16.1"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:16.2",
			"fix_state": "Affected",
			"package_name": "python-django20",
			"product_name": "Red Hat OpenStack Platform 16.2"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:17.1",
			"fix_state": "Affected",
			"package_name": "python-django",
			"product_name": "Red Hat OpenStack Platform 17.1"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:18.0",
			"fix_state": "Affected",
			"package_name": "python-django",
			"product_name": "Red Hat OpenStack Platform 18.0"
		},
		{
			"cpe": "cpe:/a:redhat:satellite:6",
			"fix_state": "Affected",
			"package_name": "python-django",
			"product_name": "Red Hat Satellite 6"
		},
		{
			"cpe": "cpe:/a:redhat:storage:3",
			"fix_state": "Affected",
			"package_name": "python-django",
			"product_name": "Red Hat Storage 3"
		},
		{
			"cpe": "cpe:/a:redhat:rhui:4::el8",
			"fix_state": "Affected",
			"package_name": "python-django",
			"product_name": "Red Hat Update Infrastructure 4 for Cloud Providers"
		}
	],
	"csaw": false
}
